### PR TITLE
docs(codes): Android Support: Basic Controls and Input + Display

### PR DESCRIPTION
**Pull request by KingCoinless**
### Android version/build?
Android 10
### What codes did you test?
C_BRI_UP
C_BRI_DN
C_BRI_MIN
C_BRI_MAX
C_BRI_AUTO
C_BKLT_TOG
C_ASPECT
C_SLOW
C_SLOW2
C_SNAPSHOT
LANG2
### How did you test?
Manual use with my Helix split keyboard using the associated ZMK shield. Typed in a note file and browser when testing `LANG2` functionality. Used the “Play Music”, “Vanilla Music”, and ‘YouTube’ apps for testing `C_SLOW` and `C_SLOW2`.
### Have you any useful information?
Most of these commands do not work in Android. `C_SNAPSHOT` switches to the camera and takes a picture. I could only use `C_BRI_UP`, and not other brightness options. Contrary to existing documentation, `LANG2` was fully functional for me as a language toggle between English and Japanese input.
### Has anyone else tested/verified it?
Only `LANG2` has been tested and marked as non-functional. My results concluded that LANG2 was in fact usable in Android for switching between Japanese and English input (like a toggle), contradicting existing docs.

### CODE REVIEW

**Review by caksoylar — APPROVED** [review]
Looks good to me based on the code changes and the deploy preview. I have not personally tested the keycodes on Android.

File: docs/src/data/hid.js [hunk 1]
```diff
@@ -5184,7 +5184,7 @@ export default [
     os: {
       windows: null,
       linux: true,
-      android: null,
+      android: false,
```
Comment (dxmh) [inline docs/src/data/hid.js:5187]: Should we seek a second verification for this one before listing `C_BRIGHTNESS_DEC` as unsupported on Android?

It seems strange that `C_BRIGHTNESS_INC` does work but not `C_BRIGHTNESS_DEC`.

I noticed in https://github.com/zmkfirmware/zmk/pull/600#discussion_r776129455 that `C_BRIGHTNESS_DEC` didn't work for Windows during initial testing a year ago, but that it did work in subsequent testing more recently. (So perhaps something changed under the hood that may have resolved this?)

Comment (KingCoinless) [reply docs/src/data/hid.js:5187]: It also seems reasonable to me to have someone check this. If someone wants to test it and confirm this functionality, I'll commit a suggested change.

Comment (cgoates) [reply docs/src/data/hid.js:5187]: FWIW, on Android 10 neither `C_BRIGHTNESS_INC` nor `C_BRIGHTNESS_DEC` work for me.

**Comment by petejohanson** [comment]
Can we retest this w/ the latest consumer page refactors?

File: docs/src/data/hid.js (hunk 1 above)
Comment (Nick-Munnich) [inline docs/src/data/hid.js:5187]:
```suggestion
      android: true,
```
Tested and found to be working. If I can't commit this suggestion I'll open a new PR.

File: docs/src/data/hid.js [hunk 2]
```diff
@@ -3590,7 +3590,7 @@ export default [
     os: {
       windows: null,
       linux: true,
-      android: false,
+      android: true,
```
Comment (Nick-Munnich) [inline docs/src/data/hid.js:3593]:
```suggestion
      android: false,
```
Tested and found that this does not work on my device (Android 14)

**Comment by Nick-Munnich** [comment]
Tested and found C_BRIGHTNESS_INC, C_BRIGHTNESS_DEC, C_SNAPSHOT to be working, while all others are not working.

**Pull request merged**